packages/devtools_app/lib/src/memory/memory_screen.dart

```diff
@@ -287,9 +286,9 @@ class MemoryBodyState extends State<MemoryBody>
     });
 
     addAutoDisposeListener(controller.androidCollectionEnabled, () {
+      final isAndroidCollection = controller.androidCollectionEnabled.value;
```

Comment:
Sorry if I was unclear - I meant local to this class, not the callback context. Then you'll be able to use that local variable in the build method below.

Reply:
Done.